### PR TITLE
WCP-1142 support for udp video streaming

### PR DESCRIPTION
**Pull request by pawcio16m**

### CODE REVIEW

File: messagegenerator.cpp [hunk 1]
```diff
@@ -176,6 +188,13 @@ QJsonObject MessageGenerator::createMeterCounterStatusIndPayload(QString val, co
     return payload;
 }
 
+QJsonObject MessageGenerator::createStartVideoStreamingPayload(int port)
+{
+    QJsonObject payload;
```
Comment (rafalgliniak) [inline messagegenerator.cpp]: consider
```
return QJsonObject{
    {"port", port}
};
```

File: CommonItems/VideoMessageBox.qml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            topMargin: Design.mediumMargin
+            left: parent.left
+            leftMargin: Design.mediumMargin
+        }
+
+        Label { text: "Port:"}
+        TextField {
+            id: port
+            selectByMouse: true
+            text: "5000"
```
Comment (rafalgliniak) [inline CommonItems/VideoMessageBox.qml]: Is it possible to keep impl details (default port in this case) in backend?

File: messagegenerator.h [hunk 3]
```diff
@@ -73,6 +75,7 @@ public slots:
     QJsonObject createHeader(const QString& msgName, const QString& msgType, MessageId messageId);
     QJsonObject createObjectStatusIndPayload(const QString& obj, const QString& val);
     QJsonObject createMeterCounterStatusIndPayload(QString val, const QString& unit);
+    QJsonObject createStartVideoStreamingPayload(int port);
```
Comment (rafalgliniak) [inline messagegenerator.h]: const method

File: CommonItems/VideoMessageBox.qml (hunk 2 above)
Comment (pawcio16m) [reply CommonItems/VideoMessageBox.qml]: yes, changed

**Review by rafalgliniak — APPROVED** [review]

**Pull request merged**